### ISSUE TITLE
I can not run Prisma studio

### ISSUE THREAD

**Issue by tanggyyggnat** [prisma/studio#1169]
### Bug description

I have run 
- npx prisma generate
- npm run dev
- and then npx prisma studio
but Fatal Error

### Environment & setup

- OS: 
- Browser: 
- Database:

**Comment by chrishoermann** [comment]
since updating to the latest version I cannot start prisma studio too:

Browser says: ERR_EMPTY_RESPONSE localhost did not send any data

**Comment by yigithanyucedag** [comment]
Same here
ERR_EMPTY_RESPONSE

**Comment by aviciida** [comment]
I'd been in the same situation where I got the same error (ERR_EMPTY_RESPONSE), but fixed it by restarting my MacBook!

**Comment by reesz** [comment]
Did anybody of you fix it? Restarted everything but still getting ERR_EMPTY_RESPONSE from the browser.
Prisma Studio seems to start fine:
```
> npx prisma studio

Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Prisma Studio is up on http://localhost:5555
```
But can't be accessed. It just worked a minute ago.
Only thing I did in between was install Android Studio. 🤔

**Comment by LeRomino** [comment]
Same for me:
ERR_EMPTY_RESPONSE

and the solution that works for me is to restart the computer

**Comment by LeRomino** [comment]
I have a feeling that the simulator (Android Studio) has something to do with it

**Comment by ivan-acosta08** [comment]
> I have a feeling that the simulator (Android Studio) has something to do with it

Thats right, i close the simulator and everything works now, i dont know why

**Comment by ChangNoi69** [comment]
Yes, closed the android simulator and it works now again.

**Comment by MrGiddz** [comment]
Yes, closing android simulator seems to fix it... weird.

**Comment by Ryguy10712** [comment]
Yes. Android emulator and prisma studio use the same port. Just launch the android emulator manually with the -port argument to specify a custom port.

Open your terminal.
Navigate to the directory where your Android SDK is located. Usually, it's in the `~/Library/Android/sdk` directory for macOS and `C:\Users\Your-Username\AppData\Local\Android\Sdk` for Windows.
Go to the emulator directory.
List all your available emulators by running the command `emulator -list-avds`.
Start your emulator by running the command `emulator -avd Your_Avd_Name -port 5556`.

happy coding :D

**Comment by meidikawardana** [comment]
Shouldn't we start petition to change the default port of prisma?

**Comment by Ryguy10712** [comment]
Almost a year later lol. But I learned something way more helpful and thought I'd share.

Disregard everything I said about launching the android emulator on a custom port. It's unnecessarily complicated (and even a little disheartening) when most development environments will automatically boot the emulator on the default port without making you play around with verbose console commands or Android Studio. 

You can actually specify the Prisma Studio port with -p or --port!

`npx prisma studio -p 0001`

**Comment by alizaeda** [comment]
yeah after updating the port `--port 5556` it worked

**Comment by igalklebanov** [comment]
Hey :wave:

Prisma ORM 7 introduces a brand new standalone [Studio](https://www.prisma.io/docs/orm/tools/prisma-studio), built from the ground up to work everywhere, with or without Prisma ORM. Consequently, all "old Studio" issues are being closed as there won't be any further work on it. :mega:

The new Studio is also available in the Prisma VS Code extension and in the Prisma Console application. :electric_plug:

Help shape, improve and fix the new Studio with us in this repository by submitting bug reports and feature requests. :construction_worker:
